wdl2cwl/tests/cwl_files/seqtk.cwl

```diff
@@ -0,0 +1,39 @@
+class: CommandLineTool
+id: Sample
+inputs:
+  - id: sequenceFile
+    type: File
+  - id: fractionOrNumber
+    type: float
+  - id: outFilePath
+    default: subsampledReads.fq.gz
+    type: string
+  - id: twoPassMode
+    default: false
+    type: boolean
+  - id: zip
+    default: true
+    type: boolean
+outputs:
+  - id: subsampledReads
+    type: File
+    outputBinding:
+        glob: outFilePath
+requirements:
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            set -e -o pipefail
+            mkdir -p "$(dirname $(inputs.outFilePath))"
+            $(inputs.preCommand)
```

Comment:
Have you tried running this CWL description yet?

Reply:
No. I have only run CollectQualityYieldMetrics.cwl yet.

Reply:
okay. Please test it. I think the `$(dirname ` might be mistaken for a CWL expression or parameter reference and may need to be escaped